extensions/binding/org.eclipse.smarthome.binding.tradfri/ESH-INF/thing/thing-types.xml

```diff
@@ -6,6 +6,7 @@
 
     <bridge-type id="gateway">
         <label>Trådfri Gateway</label>
+        <description>IKEA TRÅDFRI IP Gateway</description>
```

Comment:
We should decide whether we use capitals for TRåDFRI or not - label & desc should be in sync.